### ISSUE TITLE
Incorrect formatting of variable initialiser that sets reference

### ISSUE THREAD

**Issue by RogerChristopher** [Barteling/STweep#25]
If I reformat the following code using the attached settings...
```
    FlowPackConfig : REFERENCE TO ST_FlowPackConfig REF= HmiInterface.MachineConfig.LH_Config;
    FlowPackRecipe : REFERENCE TO ST_FlowPackRecipe REF= HmiInterface.ActiveRecipe.LH_Wrapper;
```
I get...
```
    FlowPackConfig : REFERENCE TO ST_FlowPackConfig REF= HmiInterface.
    MachineConfig.;
    FlowPackRecipe : REFERENCE TO ST_FlowPackRecipe REF= HmiInterface.
    ActiveRecipe.;
```

[Settings.zip](https://github.com/Barteling/STweep/files/6001537/Settings.zip)

**Comment by Barteling** [comment]
That doesnt look good, thanks for reporting.  i'll fix it asap

**Comment by Barteling** [comment]
@RogerChristopher  I fixed this in 3.0.2-Beta
https://www.stweep.com/beta-releases/
